### ISSUE TITLE
More useful error when running as non-root?

### ISSUE THREAD

**Issue by schoen** [certbot/certbot#2734]
I tried running the client as non-root by mistake and it failed with

```An unexpected error occurred:
IOError: [Errno 13] Permission denied: '/var/log/letsencrypt/letsencrypt.log'
Please see the logfile 'letsencrypt.log' for more details.```

Maybe it could say something more useful about needing to be run as root in this case.

**Comment by bmw** [comment]
I agree. I noticed this when preparing my RightsCon demo. I was very careful to always run the client with `sudo` during the demo to avoid this embarrassingly bad error message.

**Comment by whitenoiseoss** [comment]
I'm going to have a look at this this weekend.

**Comment by bmw** [comment]
Fixed in #3265.
